x-pack/plugins/maps/public/connected_components/toolbar_overlay/tools_control/tools_control.test.tsx

```diff
@@ -30,7 +33,7 @@ test('renders', async () => {
 });
 
 test('Should render cancel button when drawing', async () => {
-  const component = shallow(<ToolsControl {...defaultProps} isDrawingFilter />);
+  const component = shallow(<ToolsControl {...defaultProps} disableToolsControl={true} />);
```

Comment:
This still does not capture the test case. Please change the following
1) In default props, set filterModeActive to false.
2) In "Should render cancel button when drawing" test case, set filterModeActive to true.